cmd/fetch_policy.go

```diff
@@ -0,0 +1,80 @@
+// Copyright 2022 Red Hat, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+//////////////////////////////////////////////////////////////////////////////
+//
+// Define the `ec fetch policy <source-url>` command.
+//
+// As it says in the docs it's not generally needed for the EC contract, but
+// could be useful generally for hacking and troubleshooting purposes.
+//
+// If/when we designate some commands as "experimental" or "plumbing" then this
+// could be one of them.
+//
+//
+package cmd
+
+import (
+	"path/filepath"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+
+	"github.com/hacbs-contract/ec-cli/internal/downloader"
+	"github.com/hacbs-contract/ec-cli/internal/utils"
+)
+
+func fetchPolicyCmd() *cobra.Command {
+	var destDir string = "policy"
+	var useWorkDir bool = false
+
+	cmd := &cobra.Command{
+		Use:   "policy <source-url>",
+		Short: "Fetch policy rules from a git repository or other source",
+		Long: `Fetch policy rules (rego files) from a git repository or other source.
+
+This command is actually a thin wrapper for 'conftest pull' so you can refer to the
+conftest pull documentation for more details on the different types of supported source
+URLs.
+
+This is not typically required to verify the Enterprise Contract, but it's exposed for
+troubleshooting and debugging purposes.
+`,
```

Comment:
nit: cobra will add a new line, so this will end up looking like an extra line. Consider removing it.